File: ReverseNoTill1.java
import java.util.*;

class DisplayReverse
{
    public void Reverse(int No)
    {
        int iCnt = 0;

        System.out.println("Reverse numbers are:");
        for(iCnt = No; iCnt >=1; iCnt--)
        {
            System.out.print(iCnt+"\t");
        }
    }
}

class ReverseNoTill1
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        int iNo = 0;

        System.out.println("Enter the number: ");
        iNo = sobj.nextInt();

        DisplayReverse dobj = new DisplayReverse();

        dobj.Reverse(iNo);

    }
}

File: ReverseNoTill1.c
#include<stdio.h>

void DisplayReverse(int No)
{
    int iCnt = 0;

    printf("Reverse Numbers are:\n");
    for(iCnt = No; iCnt >= 1; iCnt--)
    {
        printf("%d\t",iCnt);
    }
}

int main()
{
    int iNo = 0;

    printf("Enter the number:\n");
    scanf("%d",&iNo);

    DisplayReverse(iNo);

    return 0;
}